apps/nowcasting-app/components/charts/remix-line.tsx

```diff
@@ -39,20 +39,23 @@ export type ChartData = {
   "4HR_PAST_FORECAST"?: number;
   DELTA?: number;
   DELTA_BUCKET?: DELTA_BUCKET;
+  PROBABILISTIC_UPPER_BOUND?: number;
+  PROBABILISTIC_LOWER_BOUND?: number;
   PROBABILISTIC_RANGE?: Array<number>;
```

Comment:
Can remove this prop I think?

Reply:
Thanks for the feedback, @braddf! So I've left these props in, because they are used in the `use-format-chart-data` hook, and I think we'll get a Typescript error if removed. I commented them out to check, and they were underlined in red in the chart data hook function.

Reply:
Ah gotcha, cool all good then!